docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -22,7 +22,7 @@ auth_service:
 <Admonition type="warning">
 Disabling session recording at the cluster level applies to all types of
 recordings. For example, this configuration would prevent Teleport from
-recording SSH sessions as well as desktop sessions.
+recording SSH and Kubernetes sessions as well as desktop sessions.
```

Comment:
This is awkward - we say _all types of recordings_ and then we mention 3 specifically. What about app or DB recordings?

Personally, I would have left this as-is.